compiler/plugins/input/Torch/torch-iree/InputConversion/test/torch_to_iree.mlir

```diff
@@ -27,3 +27,28 @@ module {
   util.global private @_params.classifier.weight {noinline} : tensor<30x20xf32>
   util.global private @_params.classifier.bias {noinline} : tensor<30xf32>
 }
+
+// -----
+
+// Verify we can decompose complex ops
```

Comment:
I think I'm missing something: I would have expected there to be some complex ops to decompose in this test case?

Reply:
empty_strided is part of the decomposeComplexOps pass. If you want I can add some other examples from this pass. Perhaps "complex" is a bit of a misnomer?

Reply:
Ah, I had it stuck in my head that I was expecting "complex" tensor types. Thanks - just badly aliased in my head.